<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSCConfiguration</class>
 <widget class="QWidget" name="OSCConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="host_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="host_edit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>201</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="port_chooser">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>7771</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="raw_toggle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Raw Wave</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="eeg_toggle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send EEG data</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="attmed_toggle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Attention/Meditation</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="signal_toggle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Poor Signal</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="osc_enable_toggle">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Enabled</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
